loopia.go

```diff
@@ -0,0 +1,35 @@
+package loopia
+
+import (
+  "github.com/kolo/xmlrpc"
+)
+
+const API_URL string = "https://api.loopia.se/RPCSERV"
+
+type API struct {
+    APIUsername string
+    APIPassword string
+    RCPEndpoint string
+}
+
+func newClient() (*API, error) {
+  api := &API{
+    RCPEndpoint: API_URL,
+  }
+
+  return api, nil
+}
+
+func (api *API) XmlRpcClient() *xmlrpc.Client {
+  client, _ := xmlrpc.NewClient(api.RCPEndpoint, nil)
+  return client
+}
+
+func New(username string, password string) (*API, error) {
```

Comment:
Is there a reason `New` doesn't just `return &API{...}`?

Reply:
Not really, refactored :-) Thanks for your comment